src/components/drawing/index.js

```diff
@@ -203,42 +257,35 @@ drawing.fillGroupStyle = function(s) {
             shape.call(Color.fill, d[0].trace.fillcolor);
         }
     });
-};
+}
 
 var SYMBOLDEFS = require('./symbol_defs');
 
-drawing.symbolNames = [];
-drawing.symbolFuncs = [];
-drawing.symbolNeedLines = {};
-drawing.symbolNoDot = {};
-drawing.symbolNoFill = {};
-drawing.symbolList = [];
-
 Object.keys(SYMBOLDEFS).forEach(function(k) {
     var symDef = SYMBOLDEFS[k];
-    drawing.symbolList = drawing.symbolList.concat(
+    module.exports.symbolList = module.exports.symbolList.concat(
```

Comment:
Can't you just define `symbolList` above the `module.exports = {/* */}` and use `symbolList` instead of `module.exports.symbolList`?

Reply:
The short answer ifs`No`; since the `symbolList` and friends exists as local variables too.

Reply:
Hmm. I think I could fix that one too.